modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder/form_builder.js

```diff
@@ -805,6 +817,7 @@ AUI.add(
 						var instance = this;
 
 						instance._eventHandlers.push(
+							A.one('.' + CSS_LAYOUT_BUILDER_CONTAINER).delegate('hover', A.bind('onHoverColumn', instance), '.col', instance),
```

Comment:
hey @alinedoleron , why is this repeated?

Reply:
Weird, I did't even use this line on my fix. I'd duplicated by mistake. I'll remove it.